XMonad/Actions/CopyWindow.hs

```diff
@@ -76,18 +76,21 @@ import qualified XMonad.StackSet as W
 -- "XMonad.Doc.Extending#Editing_key_bindings".
 
 -- $logHook
--- To distinguish workspaces containing copies of the focused window use
--- something like:
---
--- > sampleLogHook h = do
--- >    copies <- wsContainingCopies
--- >    let check ws | ws `elem` copies = pad . xmobarColor "red" "black" $ ws
--- >                 | otherwise = pad ws
--- >    dynamicLogWithPP myPP {ppHidden = check, ppOutput = hPutStrLn h}
--- >
--- > main = do
--- >    h <- spawnPipe "xmobar"
--- >    xmonad def { logHook = sampleLogHook h }
+--
+-- To distinguish workspaces containing copies of the focused window, use the functions below.
+-- First use 'copiesPP' that takes a pretty printer and makes it aware of copies.
+-- Then use 'copiesPP' with 'XMonad.Hooks.StatusBar.statusBarProp' :
```

Comment:
I'm not sure I follows these instructions very well; first use `copiesPP` and then use it again?  Do you want to say first build a `StatusBarConfig` with `copiesPP` and then use _that_ with `statusBarProp`?

Reply:
Thanks I have updated that. Let me generate the doc again to see if it looks OK.